extensions/3DTILES_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1623 @@
+<!-- omit in toc -->
+# 3DTILES_metadata Extension
+
+**Version 1.0.0**, February 25, 2021
+
+<!-- omit in toc -->
+## Contributors
+
+* Sean Lilley, Cesium
+* Peter Gagliardi, Cesium
+* Sam Suhag, Cesium
+* Patrick Cozzi, Cesium
+* Bao Tran, Cesium
+
+<!-- omit in toc -->
+## Status
+
+Draft
+
+<!-- omit in toc -->
+## Dependencies
+
+Written against the 3D Tiles 1.0 specification.
+
+Adds new functionality to the [`3DTILES_implicit_tiling` extension](../../3DTILES_implicit_tiling/0.0.0). See [Implicit Tile Metadata](#implicit-tile-metadata).
```

Comment:
When the novice reads top down, it seems like this extension may _only_ add new functionality to `3DTILES_implicit_tiling` since that is the first thing they read.  One more sentence may be helpful.

Let's also discuss/confirm how extensions can extend/require other extensions and make sure 3D Tiles is aligned with the Khronos and glTF approach.